src/MSBuild/TerminalLogger/TerminalLogger.cs

```diff
@@ -235,7 +236,12 @@ private void BuildStarted(object sender, BuildStartedEventArgs e)
 
         _buildStartTime = e.Timestamp;
 
-        Terminal.Write(AnsiCodes.SetProgressIndeterminate);
+        // dotnet/msbuild#8958: iTerm2 treats ;9 code to post a notification instead,
```

Comment:
Nit: I prefer full URLs
```suggestion
        // https://github.com/dotnet/msbuild/issues/8958: iTerm2 treats ;9 code to post a notification instead,
```